reference/Makefile

```diff
@@ -0,0 +1,2 @@
+genes.gff:
+	gffread -F --keep-exon-attrs genes.gtf > genes.gff
```

Comment:
Do we really need the Makefile?
It's seems quite simple, and I believe the command line is almost entirely in the header of the gff.